mods/Windows/OnlineCTR/Network_PC/Server/SV_main.c

```diff
@@ -294,39 +294,56 @@ void ProcessReceiveEvent(ENetPeer* peer, ENetPacket* packet) {
 			struct SG_MessageName* s = &sgBuffer[0];
 			struct CG_MessageName* r = recvBuf;
 
-			s->numClientsTotal = ri->clientCount;
+			BadWordList badWordList;
+			InitializeBadWordList(&badWordList);
```

Comment:
since this is constant, you can intialize it in the main function, instead of intializing it each time CG_NAME message is received

you could move badWorList outside of this scope and declare it as a global variable